src/js/helpers/apiRequest.js

```diff
@@ -4,15 +4,16 @@
  **/
 
 import Axios, { CancelToken } from 'axios';
-import kGlobalConstants from 'GlobalConstants';
 
 const mockUrl = `http://localhost:5000/api/`;
 const localUrl = `http://localhost:8000/api/`;
 
 const getBaseUrl = (params) => {
     if (params.isMocked) return mockUrl;
     if (params.isLocal) return localUrl;
-    return kGlobalConstants.API;
+    // return kGlobalConstants.API;
+    // DEV-5242: Can find good awards via advanced search.
+    return 'https://qat-api.usaspending.gov/api/';
```

Comment:
Looks like this was a temporary change for testing (not meant to be committed)?

Reply:
Yes!